packages/frontend-2/components/workspace/invite/Banners.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div class="flex flex-col divide-y divide-outline-3">
+    <WorkspaceInviteBanner v-for="item in items" :key="item.id" :invite="item" />
+  </div>
+</template>
+<script setup lang="ts">
+import { graphql } from '~~/lib/common/generated/gql'
+import type { WorkspaceInviteBanners_UserFragment } from '~~/lib/common/generated/gql/graphql'
+
+/**
+ * TODO: Add this to new dashboard page and remove from projects dashboard
```

Comment:
Currently all banners show up in projects page (current dashboard), but they should be moved to the new dashboard once its in

Reply:
The new dashboard is in at /dashboard/. But this can be moved in another ticket as you say

Reply:
I've added the banners there as well